src/main/java/BaseEmployee.java

```diff
@@ -0,0 +1,31 @@
+import java.time.YearMonth;
+
+abstract public class BaseEmployee {
+
+    String name;
+    String surname;
+    int baseSalary;
+    private int yearOfEmployment;
+    final int DEFAULT_BASE_SALARY = 3000;
```

Comment:
pole o takiej nazwie powinno być static

Reply:
Dodatkowo, można by stworzyć konstruktor klasy który nie przyjmuje atrybutu baseSalary, a w którym tą wartość będziemy przypisywać od odpowiedniego pola

Reply:
Poprawione , stworzyłem drugi konstruktor, ale nie jestem pewien czy o to do końca chodziło